src/OrbitGl/App.cpp

```diff
@@ -1318,7 +1318,9 @@ void OrbitApp::StartCapture() {
   bool enable_introspection = IsDevMode() && data_manager_->get_enable_introspection();
   bool enable_user_space_instrumentation =
       IsDevMode() && data_manager_->enable_user_space_instrumentation();
-  double samples_per_second = data_manager_->samples_per_second();
+  constexpr double kSamplesPerSecondDefault = 1000.0;
```

Comment:
So this is a complete nitpick, but `kCallstackSamplingPeriodMsDefaultValue` in mainwindow.cpp is basically the same, this could be merged. Anyways, feel free to ignore.

Reply:
What would be the best place for the shared constant? Maybe `DataManager`?
Either way, I'm going to leave this as this should just be temporary until the setting is also expose outside of `--devmode`.